someSong/src/pages/home/home.html

```diff
@@ -12,40 +12,77 @@
 </ion-header>
 
 <ion-content class="no-scroll" padding="5px">
-  <ion-list-header>
-    My Questions
-  </ion-list-header>
-  <ion-scroll scrollY="true" class="myQuestions">
-    <ion-list no-lines no-padding *ngIf="userQuestions">
-      <button ion-item *ngFor="let question of userQuestions | keyValueFilter" (click)="goToQuestion(question.key)">
-        {{question.value.title}}
-        <ion-icon name="checkmark" item-left color="secondary" *ngIf="question.value.correctAnswer"></ion-icon>
-        <ion-icon name="close" item-left color="danger" *ngIf="!question.value.correctAnswer"></ion-icon>
+  <div class="scrollableContent">
+    <ion-list-header class="list-header">
+      My Questions
+    </ion-list-header>
+    <ion-scroll scrollY="true">
+      <ion-list no-lines no-padding *ngIf="userQuestions && !isEmpty(userQuestions) && !questionLoading">
```

Comment:
1. Why check both `userQuestions` and `isEmpty(userQuestions)`?
2. Also, in line 27 you checked ` *ngIf="userQuestions..."` did you mean - ` *ngIf="!userQuestions..."`?
3. And also - the result of ` *ngIf="userQuestions"` is always **TRUE** regardless whether the user has questions or not (because it is initialized as an object).